pkg/api/handlers/compat/images_build.go

```diff
@@ -375,6 +375,13 @@ func BuildImage(w http.ResponseWriter, r *http.Request) {
 		ulimits = m
 	}
 
+	// if layers field not set assume its not from a valid podman-client
+	// could be a docker client, set `layers=true` since that is the default
+	// expected behviour
+	if _, found := r.URL.Query()["layers"]; !found {
```

Comment:
Can you move setting the default value up to line 128?

Reply:
@vrothberg thanks moved it with other defaults.

Reply:
I don't see code changes.  Did you miss adding them to the commit?

Reply:
@vrothberg moved it just after we decode `query` struct and where we start adding other defaults. i.e after line no `135`.

Reply:
The other parameters are more complex types.  It'll work for a boolean (see `Rm`).

Reply:
@vrothberg oh yes correct pushed a new version. Thanks.